projects/gubernator/create-prerequisites.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+secret=$( mktemp )
+cat <<EOF >secret
+{
+    "$( oc get route gubernator -o "jsonpath={.spec.host}" )": {
+        "session": "$( openssl rand -base64 16 )",
+        "github_client": {
+            "id": "${GUBERNATOR_CLIENT_ID}",
```

Comment:
@smarterclayton @dobbymoodge we need an oauth client for this -- @dobbymoodge it's going to ask for permissions to touch GitHub on your behalf in order to build the dashboard